Resources/Textures/Clothing/Uniforms/uniform_detective.rsi/meta.json

```diff
@@ -0,0 +1,191 @@
+﻿{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from https://github.com/tgstation/tgstation/",
```

Comment:
This is just a nitpick, but you should probably specify the commit at which you took these sprites. Same with the other RSI you added. Just in case.

Reply:
whoops. added for all 4